services/secure-gateway/policyStorage/policyStorage.go

```diff
@@ -100,10 +101,15 @@ func New(workDir string, config *appConfig.PolicyStorage) (result persist.Adapte
 		log.Panicln("Minio bucket with casbin policies doesn't exist")
 	}
 
-	filePath := path.Join(workDir, config.CasbinModel)
-	error := minioClient.FGetObject(config.MinioBucketName, config.CasbinModel, filePath, minio.GetObjectOptions{})
+	filePath := path.Join(workDir, config.CasbinPolicy)
```

Comment:
use policy library, listen from event from nats in the future

Reply:
Added task #756